spellmonitor.lic

```diff
@@ -110,8 +110,10 @@ spell_action = proc do |server_string|
   end
 
   # The start of one or more active spells!
-  if data =~ %r{^<pushStream id="percWindow"/>}
+  if data =~ %r{<pushStream id="percWindow"/>}
     DRSpells.grabbing_spell_data = true
+    index = data.index("<pushStream id=\"percWindow\"/>")
+    data.slice!(index, index+29).strip!
     data.slice!(0, 29).strip!
```

Comment:
I think you meant to replacel ine 117 with `data.slice!(index, index+29).strip!`
```suggestion
```